Services/TimesheetsExporter.php

```diff
@@ -46,6 +46,10 @@ public function generateData(array $criteria): array
             $row['fullname'] = $row['firstname'] . ' ' . $row['lastname'];
         }
 
+        foreach ($data as &$row) {
+            $row['workDate'] =  format($row['workDate'])->date();
+        }
```

Comment:
Merge the body of this loop into the already existing `foreach` loop (and remove the extra ` ` (space) before `format(`).